### PR TITLE
[14.0][ADD] partner_delivery_zone_zip

### PR DESCRIPTION
**Pull request by rousseldenis**
@baimont @sergio-teruel

### CODE REVIEW

**Review by flotho — APPROVED** [review]
successfully tested on runBOAT (BTW it rocks!)
Thanks for the job
LGTM

File: partner_delivery_zone_zip/demo/partner_delivery_zone.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo noupdate="1">
+
+    <!-- TODO
+    <record model="partner.delivery.zone" id="partner_delivery_zone_demo_1">
+       <field name="name">...</field>
+    </record>
+    -->
+
+</odoo>
```
Comment (sergio-teruel) [inline partner_delivery_zone_zip/demo/partner_delivery_zone.xml]: Remove ??

File: partner_delivery_zone_zip/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+class ResPartner(models.Model):
+
+    _inherit = "res.partner"
+
+    @api.onchange("zip")
+    def onchange_zip(self):
+        delivery_zone_zip = self.env["partner.delivery.zone.zip"].search(
+            [("name", "=", self.zip)], limit=1
+        )
+        self.delivery_zone_id = delivery_zone_zip.delivery_zone_id
```
Comment (sergio-teruel) [inline partner_delivery_zone_zip/models/res_partner.py:16]: Could this field be computed writable?

File: partner_delivery_zone_zip/views/partner_delivery_zone.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 ACSONE SA/NV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="partner_delivery_zone_form_view">
+        <field
+            name="name"
+        >partner.delivery.zone.form (in eri_partner_delivery_zone_zip)</field>
```
Comment (sergio-teruel) [inline partner_delivery_zone_zip/views/partner_delivery_zone.xml]: eri_partner_delivery_zone_zip typo error??

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Pull request closed without merging**